sparse/mlir_backend/_constructors.py

```diff
@@ -111,7 +110,7 @@ class Dense(ctypes.Structure):
 
             def to_np(self) -> np.ndarray:
                 data = self.data.to_numpy()[: self.data_len]
-                return data.reshape((self.shape_x, self.shape_y))
+                return data.copy().reshape((self.shape_x, self.shape_y))
```

Comment:
Ugh, copies are slow. I was trying to avoid this footgun.

Reply:
Closing as we discussed it in a call.